keymapper/user.py

```diff
@@ -58,8 +58,18 @@ def get_user():
     return user
 
 
+def get_home(user):
+    """Try to find the user's home directory."""
+    try:
+        return pwd.getpwnam(user).pw_dir
+    except:
```

Comment:
is there reason to believe that this might fail? i.e. are the except and the fallback return statement below needed?

Reply:
Not that I'm aware of but I'm afraid it might and decided to keep the old behavior (just didn't want to break your code). I'm okay removing it.

Reply:
yeah lets remove it then

Reply:
Done